### DISCUSSION TITLE
More groups for STM32

### DISCUSSION THREAD

**Discussion by atlesg** [luni64/TeensyStep#135] [Q&A]
Hi luni64,

Your library is amazing, I have modified a bit to your stm32 folder and was able to control motors using F1 and F4. Thing is, what should I do if I want to extent from group of 4 controllers to more? suggesting that STM has more spare timers to use, and we need to how-to-do.

Please suggest me where should I start (which file, which part of code that I should modify...). I think lots of people are in need of this too :D 

Thanks,
Atlesg

**Comment by quartit** [comment]
Hello, 
I see that you managed to install the teensystep lib on a STM32 board. 
Can you explain me what to do with the include of arduino header ?
How can I install this lib in stm32IDE ?
Thanks

**Reply by atlesg** [reply]
Hi! I did not use this lib on STM32CubeIDE but Visual Studio Code with PlatformIO and Arduino framework. I have tried STm32CubeIDE but did not succeed :<
